app/models/manageiq/providers/cloud_manager/provision_workflow.rb

```diff
@@ -141,4 +142,16 @@ def provider_or_tenant_object
     obj = src[:cloud_tenant] || src[:ems]
     load_ar_obj(obj)
   end
+
+  def openstack?(networks)
+    networks.select do |cloud_network|
+      cloud_network.class.ancestors.include?("ManageIQ::Providers::Openstack::NetworkManager::CloudNetwork")
```

Comment:
core code can't have any mentions of OpenStack, do this via support mixin or use specific allowed_cloud_networks in OpenStack plugin